src/unix/fs.c

```diff
@@ -62,6 +63,43 @@
 
 #if defined(__APPLE__)
 # include <copyfile.h>
+# include <sys/attr.h>
+
+void uv__prepare_setattrlist_args(uv_fs_t* req,
+                                  struct attrlist* attr_list,
+                                  struct timespec* times,
```

Comment:
If you specify this as `struct timespec (*times)[3]` and pass it as `&times` at the call site, then callers can't pass in a wrong-sized array.

Aside: this function should be `static`.

Aside 2: I'd `*size = 0` first so callers don't have to (and can't forget.)